### PR TITLE
interfaces/builtin: allow MM to access login1

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
So it can properly shutdown devices on suspend/hibernate.

### CODE REVIEW

File: interfaces/builtin/modem_manager.go [hunk 1]
```diff
@@ -117,6 +117,19 @@ dbus (receive, send)
...
+    bus=system
+    path=/org/freedesktop/login1
+    interface=org.freedesktop.login1.Manager
+    peer=(label=unconfined),
+dbus (send)
+    bus=system
+    path=/org/freedesktop/login1
+    interface=org.freedesktop.login1.Manager
+    member=Inhibit
+    peer=(label=unconfined),
```
Comment (jdstrand) [inline interfaces/builtin/modem_manager.go]: This send rule looks fine.

File: interfaces/builtin/modem_manager.go [hunk 2]
```diff
@@ -117,6 +117,19 @@ dbus (receive, send)
     path=/org/freedesktop/ModemManager1{,/**}
     interface=org.freedesktop.DBus.*
     peer=(label=unconfined),
+
+# Allow accessing logind services to properly shutdown devices on suspend
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/login1
+    interface=org.freedesktop.login1.Manager
+    peer=(label=unconfined),
```
Comment (jdstrand) [inline interfaces/builtin/modem_manager.go]: The network-manager interface has similar rules, but limited receives to PrepareForSleep, SessionNew and SessionRemoved. What is login1.Manager sending to ModemManager?

Comment (tonyespy) [reply interfaces/builtin/modem_manager.go]: It looks like MM only listens for `PrepareForSleep`, and uses `Inhibit` to create a delay lock to give it time to tear down its connections.  It doesn't look like it's listening for any other signals from logind.

**Comment by alfonsosanchezbeato** [comment]
@jdstrand update to reflect the signals to which MM really listens: {PrepareForSleep,SessionNew,SessionRemoved}. I am not sure where the denials from Session* come, it must be some library as that is not in the MM sources.

**Comment by alfonsosanchezbeato** [comment]
CI error seems unrelated to the PR:

2018-02-14 14:12:02 Error preparing linode:fedora-27-64 :

**Pull request merged**